sql/updates/world/2016_01_XX_XX_world.sql

```diff
@@ -0,0 +1,26 @@
+-- THERE ARE A LOT OF EXCESS COMMENTS IN THIS FILE. They will be removed after code review.
+-- Add sniffed gossip texts to menu
```

Comment:
sorry, unclear comment: there are 2 "Send me to the Abyssal Shelf!" strings in `broadcast_text` and I'm not sure how to tell which is the correct one to use in this case. The other one, I presume, is used for the Horde NPC Wing Commander Brack (19401).